testng-core/src/test/java/test/retryAnalyzer/issue2684/IssueTest.java

```diff
@@ -0,0 +1,41 @@
+package test.retryAnalyzer.issue2684;
+
+import static org.testng.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import org.testng.xml.XmlClass;
+import org.testng.xml.XmlGroups;
+import org.testng.xml.XmlRun;
+import org.testng.xml.XmlSuite;
+import org.testng.xml.XmlTest;
+
+public class IssueTest {
+
+  @Test
+  public void testRetriedTestsWithGroupConfigs() {
```

Comment:
Can you please move this test into `test.retryAnalyzer.RetryAnalyzerTest` and add a `description` attribute to the `@Test` which refers to `GITHUB-2684`

Reply:
Done